news: give warning about bugtracker downtime (#275)
mixxxdj/website: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md

> title: Bugtracker temporarily unavailable

Reviewer: ```suggestion
title: Bug tracker temporarily unavailable
```

> title: Bug tracker temporarily unavailable

Reviewer 2: Title case?

authors: Nikolaus Einhauser
tags: infrastructure, development, contribute
comments: yes
status: draft

Hello Everybody.

> After much preparation, we're moving from Launchpad to Github issues. We'll

Reviewer: ```suggestion
After thorough preparation, we're moving from Launchpad bugs to GitHub issues.
```

> After thorough preparation, we're moving from Launchpad bugs to GitHub issues.

Reviewer 2: ```suggestion
After thorough preparation, we're finally moving from Launchpad bugs to GitHub issues.
```

> publish a more detailed blogpost after the migration is done. For now, We'd

Reviewer: ```suggestion
For now, We'd
```
The sentence is repeated below.

Author: I'm aware, but I wanted to mention the extra blogpost upfront. Afterwards I need it to refer to the time when its published.

Reviewer 2: I'm any case the sentence is incomplete and should be fixed or removed.

Author: fixed

> just like to let everybody know, that we will disable [our bugtracker on

Reviewer: ```suggestion
just like to let everybody know, that we will disable [our bug tracker on
```

> launchpad](https://bugs.launchpad.net/mixxx) sometime afternoon Monday the 22.

Reviewer: ```suggestion
launchpad](https://bugs.launchpad.net/mixxx) sometime at Monday the 22nd
```
Afternoon depends on the time zone.

Author: I could just include the timezone
```suggestion
launchpad](https://bugs.launchpad.net/mixxx) sometime on Monday the 22nd
```

Reviewer: Yes that is probably the best.

> of August. Any activity on Launchpad from that point on will not be carried over

Reviewer: ```suggestion
of August. Any activity on Launchpad after that point will not be carried over
```

> to Github. Any already existing link to an issue on Launchpad will still

Reviewer: ```suggestion
to GitHub. Any already existing link to an issue on Launchpad will still
```

continue to work, but you will not be able to search for them on Launchpad anymore. Once the

> migration has been completed, the more detailed blogpost will be published and

Reviewer: ```suggestion
migration has been completed, the more detailed blog post will be published and
```

> you will be able to continue to interact with the newly migrated Issues on [our

Reviewer: ```suggestion
you will be able to interact with the newly migrated issues on [our
```

> Github repository](https://github.com/mixxxdj/mixxx/issues).

Reviewer: ```suggestion
GitHub repository](https://github.com/mixxxdj/mixxx/issues).
```

> 

Reviewer: ```suggestion
In the meantime you are welcome to discuss bugs and feature requests at [Zulip](https://mixxx.zulipchat.com).
```

Author: Does it really make sense to fragment issue discussion even more?

Reviewer 2: I agree. People will just have to wait a little.

Reviewer: Ok
